package/build/LibSassBuilder.props

```diff
@@ -14,18 +14,28 @@
     <EnableDefaultSassItems Condition="'$(EnableDefaultSassItems)'==''">true</EnableDefaultSassItems>
     <DefaultSassExcludes>**/bin/**;**/obj/**;**/node_modules/**;**/logs/**</DefaultSassExcludes>
     
-    <DefaultItemExcludes Condition="'$(EnableDefaultSassItems)'=='true'">$(DefaultItemExcludes);**/*.scss;**/*.sass</DefaultItemExcludes>
+    <!-- 
+      exclude for VS - https://docs.microsoft.com/en-us/visualstudio/msbuild/visual-studio-integration-msbuild#build-solutions 
+      required from this issue (VS bug) - https://github.com/johan-v-r/LibSassBuilder/issues/15
+      -->
+    <DefaultItemExcludes Condition="'$(EnableDefaultSassItems)'=='true' and '$(BuildingInsideVisualStudio)'=='true'">
+      $(DefaultItemExcludes);**/*.scss;**/*.sass
+    </DefaultItemExcludes>
 
     <!-- Provide LibSassBuilderArgs to take complete control -->
     <LibSassBuilderArgs Condition=" '$(LibSassBuilderArgs)'=='' "></LibSassBuilderArgs>
     
     <!-- Add target to CustomCollectWatchItems -->
     <CustomCollectWatchItems>$(CustomCollectWatchItems);_LibSass_CustomCollectWatchItems</CustomCollectWatchItems>
   </PropertyGroup>
-      
+  
   <ItemGroup Condition="'$(EnableDefaultItems)' == 'true' And '$(EnableDefaultSassItems)' == 'true'">
+    <!-- add sass files that must be compiled (also gets tracked by VS) -->
     <SassFile Include="**/*.scss" ExcludeFromSingleFile="true" CopyToPublishDirectory="PreserveNewest" Exclude="$(DefaultSassExcludes);$(DefaultExcludesInProjectFolder);**/_*" />
     <SassFile Include="**/*.sass" ExcludeFromSingleFile="true" Exclude="$(DefaultSassExcludes);$(DefaultExcludesInProjectFolder);**/_*" />
+
+    <!-- add all other sass files to show up in VS -->
+    <None Condition="'$(BuildingInsideVisualStudio)'=='true'" Include="**/_*.scss;**/_*.sass" />
```

Comment:
This is actually required as part of #22 where the variable `**/_*.scss` files are hidden because of the above `DefaultItemExcludes`

Strange thing here though is that it doesn't seem to work for files in my `wwwroot` folder - any ideas welcome

Reply:
The files in `wwwroot` show when _not excluded_ in the `SassFile` above - but it needs to be excluded because don't want to pass them into LSB.